modules/learn/pages/buckets-memory-and-storage/memory.adoc

```diff
@@ -9,31 +9,33 @@
-Note that the xref:services-and-indexes/services/data-service.adoc[Data Service] must run on at least one node; and that on each of those nodes, quotas for buckets, specified at the time of bucket-creation, are subtracted from the quota allocated to the Data Service.
+You must configure your memory quota allocations for each service in Couchbase Server, except for the xref:services-and-indexes/services/query-service.adoc[Query Service] and xref:services-and-indexes/services/backup-service.adoc[Backup Service]. You can change the availability of memory resources in your cluster based on how you assign your services to each node.
 
-The memory-quota allocation specified for a given service applies to every instance of that service across the cluster.
-For example, if 2048 MB is specified for the xref:services-and-indexes/services/analytics-service.adoc[Analytics Service], and the Analytics Service is run on three of a cluster's five nodes, each of the three instances of the service is duly allocated 2048 MB.
-Note that it is not possible to have different memory allocations across multiple instances of the same service within a single cluster.
+The memory quota you allocate for a service applies to every instance of that service across your cluster. 
+For example, if you allocate 2048 MB to the xref:services-and-indexes/services/analytics-service.adoc[Analytics Service], and you run the Analytics Service on three of a cluster's five nodes, each instance of the service has 2048 MB of memory.
 
-As a guideline, Couchbase recommends that no more than 90% of a node's memory (80% on nodes with a small amount of total memory) be allocated to the server and its services. 
-The firm limits on server memory allocation are: stem:[max(total\_memory - 1, 80\% \times total\_memory)] where `total_memory` is the maximum memory on the node in GiB.
+NOTE: You can't allocate different amounts of memory for different instances of the same service in a cluster. 
 
-Instructions on how to allocate memory quotas to services when initializing a new cluster can be found in the section xref:manage:manage-nodes/create-cluster.adoc[Create a Cluster].
+Couchbase recommends that you allocate no more than 90% of a node's memory (80% on nodes with a small amount of total memory) to a server and its services. 
 
-When a node is added to a cluster, the _Default Configuration_, as established by the set-up of the first node in the cluster, is available: this covers all configurable elements, including memory quotas.
-However, if insufficient memory for the default configuration is available on the new node, the default configuration is prohibited: in such cases, settings for the new node can be custom-configured, allowing an appropriate subset of services to be specified.
+The firm limits for server memory allocation can be calculated by: stem:[max(total\_memory - 1, 80\% \times total\_memory)] where `total_memory` is the maximum memory on the node in GiB.
```

Comment:
The STEM formula breaks the line, leaving an ugly gap at the end of the previous line. Is there any way to shorten the preceding text to avoid the line break? Or put the formula in a line by itself?

Reply:
I like line by itself.